### ISSUE TITLE
wierd errors with current version

### ISSUE THREAD

**Issue by joshco** [mikekelly/hal-browser#60]
target url changed to: /
browser.js:55 target url changed to: /api/v1
URI.min.js:71 Uncaught Error: URNs do not have any generally defined hierarchical componentse.absoluteTo @ URI.min.js:71HAL.normalizeUrl @ hal.js:27(anonymous function) @ VM756:46_.each._.forEach @ underscore.js:87(anonymous function) @ VM756:4_.template.template @ underscore.js:1165HAL.Views.Links.Backbone.View.extend.render @ links.js:52HAL.Views.Resource.Backbone.View.extend.render @ resource.js:29(anonymous function) @ resource.js:8triggerEvents @ backbone.js:96Backbone.Events.trigger @ backbone.js:181$.ajax.success @ client.js:17x.Callbacks.c @ jquery-1.10.2.min.js:4x.Callbacks.p.fireWith @ jquery-1.10.2.min.js:4k @ jquery-1.10.2.min.js:6x.ajaxTransport.send.r @ jquery-1.10.2.min.js:6
browser.js:55 target url changed to: http://demo.osdi.io/api/v1
browser.html:1 XMLHttpRequest cannot load http://demo.osdi.io/api/v1. A wildcard '*' cannot be used in the 'Access-Control-Allow-Origin' header when the credentials flag is true. Origin 'http://joshco.local.host:3000' is therefore not allowed access.
